src/metabase/middleware.clj

```diff
@@ -216,10 +226,12 @@
   (when-let [k (ssl-certificate-public-key)]
     {"Public-Key-Pins" (format "pin-sha256=\"base64==%s\"; max-age=31536000" k)}))
 
-(defn- security-headers [& {:keys [allow-iframes?]
-                            :or   {allow-iframes? false}}]
+(defn- security-headers [& {:keys [allow-iframes? allow-cache?]
+                            :or   {allow-iframes? false, allow-cache? false}}]
```

Comment:
This doesn't hurt anything, but it's also not required. If `allow-cache?` isn't in the map, it will be nil which is falsey.

Reply:
I was just following what `allow-iframe?` did. Is there a reason for one to have a default but not the other?

Reply:
I missed that `allow-iframes?` was defaulted too. It doesn't look like it's needed there either. I'd say just leave it the way you have it though, no big deal there.